cap-ci/cap-release-upgrades.yaml

```diff
@@ -38,6 +40,9 @@ schedulers:
 catapult:
   uri: https://github.com/SUSE/catapult
   branch: master
+minibroker-integration-tests:
+  uri: https://github.com/SUSE/minibroker-integration-tests.git
+  branch: master
```

Comment:
And here.

Reply:
same ^

Reply:
Done https://github.com/SUSE/cap/pull/49/commits/d450c442f159461cd7bfc9262196797151610a0b